shared/actions/signup.js

```diff
@@ -57,6 +57,31 @@ function checkInviteCode (inviteCode: string): TypedAsyncAction<CheckInviteCode
   })
 }
 
+function requestAutoInvite (): TypedAsyncAction<CheckInviteCode | NavigateAppend | NavigateTo | SignupWaiting> {
+  return dispatch => new Promise((resolve, reject) => {
```

Comment:
k, i think in general this is a bad pattern, but it'll all go away w/ sagaification so /shrug